kong/runloop/handler.lua

```diff
@@ -206,6 +215,13 @@ local function register_events()
       log(DEBUG, "[events] Service updated, invalidating router")
       cache:invalidate("router:version")
     end
+
+    if data.operation == "create" or
+       data.operation == "update" then
+      if utils.hostname_type(data.entity.host) == "name" then
```

Comment:
style: a single branch instead of a nested `if` would be simpler:
```lua
if data.operation == "create" or data.operation == "update"
   and utils.hostname_type(data.entity.host) == "name"
then

end
```